### DISCUSSION TITLE
Badumdum effect

### DISCUSSION THREAD

**Discussion by kiaen1** [Drise13/PetsOptimizer#3] [General]
I see you say you don't agree with Badumdum effect, what do you mean? It seems to work in-game the way I would expect it to work, minus it working on it's own row too.

**Comment by Drise13** [comment]
Yea, when I read

>When foraging, the pets in the territory above and below contribute 1.20x Foraging Speed

I see that as only pets in territories above and below. The way the game seems to calculate this is above, below, *and self*. The self row application is not specified by the language lava used in game to describe the gene, so I didn't originally code for it that way. When I got access to that gene, my math disagreed with what I got from the game. I was confused until I did some calculations by hand and figured out that it applies to its own row as well.

Just an unintuitive description (or bug) on lava's part.

**Reply by kiaen1** [reply]
Gotcha gotcha, I wasn't sure if that was the issue or not with it. Thanks for clarifying. It's probably an issue with lavacode, so I will go ahead and add it to the bug list/see if it is intended.
